app/Models/Author.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Author extends Model
+{
+    use HasFactory;
+
+    protected $table = 'authors';
```

Comment:
No need to explicitly define the table name here. Just follow the Laravel convention of defining the model and Laravel will resolve the table name by default.

Reply:
Fix in other Models too.